flask_flatpages/flatpages.py

```diff
@@ -3,7 +3,7 @@
 flask_flatpages.flatpages
 =========================
 
-Flatpages extension.
+Flatpages extension. 
```

Comment:
Here's the issue. It afailed our (very strict!) linting due to trailing whitespace. You can run the tests locally to check things like this before commiting.

Reply:
Merged in updated - sorry my first time using git to contribute so I'm still getting my head around how to use it.